onsset/onsset.py

```diff
@@ -561,11 +561,33 @@ def condition_df(self):
         logging.info('Sort by country, Y and X')
         self.df.sort_values(by=[SET_Y_DEG, SET_X_DEG], inplace=True)
 
-    def grid_penalties(self):
+    def classify_land_cover(self, column):
+            """this is a different method employed to classify land cover and create new columns with the classification
+
+            Arguments
+            ---------
+            
+            column : list
+           
+
+            0,11 =1, 
+            6, 8 = 2
+            1, 3, 5, 12 ,13,15,=3, 
+            2,4=4, 
+            7,9,10,14,16=5 
+            """
+
+            land_cover_labels = [1,3,4,3,4,3,2,5,2,5,5,1,3,3,5,3,5]
+            bob=column.apply(lambda x:land_cover_labels[int(x)])
+            return  bob
```

Comment:
remove this line (replaced above with `return column.apply(lambda x:land_cover_labels[int(x)])`)